aquavision

```diff
@@ -1 +1 @@
-Hi Barbie
```

Comment:
❤️ His voice is warm and husky like dark melted chocolate fudge Caramel... or something.